spec/models/document_spec.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Document, type: :model do
+  it 'is valid with a name' do
+    document = Document.new(
```

Comment:
use FactoryBot
Is this PR ready for review? Pls add labels.

Reply:
yes I know. I will add FactoryBot and then add labels.